src/handlers/cloudinary.ts

```diff
@@ -0,0 +1,28 @@
+// @ts-ignore
+import { image as cloudinaryImage } from 'cloudinary/lib/cloudinary';
+
+type CloudinaryConfig = {
+  width: number;
+  height: number;
+  format: string;
+};
+
+export function signUrl(
```

Comment:
I suppose we are not gonna change any tokens during the session - can we cache this function?